.github/workflows/reduce-readme.yml

```diff
@@ -0,0 +1,34 @@
+name: Reduce README
+on:
+  push:
+    paths:
```

Comment:
this and branch is probably not needed as currently configured, it's default value

Reply:
fixed